modules/phase_field/src/auxkernels/TestEBSDAux.C

```diff
@@ -24,6 +24,11 @@ TestEBSDAux::TestEBSDAux(const InputParameters & parameters) :
 {
 }
 
+TestEBSDAux::~TestEBSDAux()
+{
+  delete _val;
```

Comment:
This is pretty annoying.  I don't really like the `EBSDReader::getPointDataAccessFunctor` interface, it just allocates memory and hands it off to the user, which is not really like other things in MOOSE and very prone to creating memory leaks, like we see here...